psydac/linalg/iterative_solvers.py

```diff
@@ -470,8 +474,8 @@ def bicg(A, At, b, x0=None, tol=1e-6, maxiter=1000, verbose=False):
         #-----------------------
         # MATRIX-VECTOR PRODUCTS
         #-----------------------
-        A .dot(p , out=v )
-        At.dot(ps, out=vs)
+        v  = A .dot(p )
+        vs = At.dot(ps)
```

Comment:
Why this change? It creates two temporary vectors for each iteration...

Reply:
yes I forgot to turn them back

Reply:
fixed